.github/workflows/ci.yaml

```
@@ -24,6 +24,8 @@ jobs:
         distribution: temurin
         java-version: "17"
         check-latest: false
+    - name: Install NDK
+      run: sudo bash -c "echo y | ${ANDROID_HOME}/cmdline-tools/latest/bin/sdkmanager --install 'ndk;26.3.11579264'"
```

I think we may as well grab the NDK version from `build.gradle.kts`, since we're liable to forget to update this version. Something like:

```suggestion
      run: |
          NDK_VERSION=$(grep -oP '(?<=ndkVersion = ")[0-9.]*' divviup/build.gradle.kts)
          yes | ${ANDROID_HOME}/cmdline-tools/latest/bin/sdkmanager --install "ndk;${NDK_VERSION}"
```
n.b. I didn't test this.